### PR TITLE
Partition R and Python package support

### PR DESCRIPTION
**Pull request by jgendrinal**

### CODE REVIEW

**Comment by enzoampil** [comment]
@jpdeleon Can you help me review these this week if you're free :smile:

**Comment by enzoampil** [comment]
`requirements.txt` is not being installed. Will have to explicitly run pip installations in `travis.yml `like with prophet ([here](https://github.com/facebook/prophet/blob/master/.travis.yml))

**Comment by enzoampil** [comment]
@jgendrinal getting this error code from the R side of TravisCI

```
Installing R
0.00s
git.checkout
1.95s$ git clone --depth=50 --branch=r-dev https://github.com/enzoampil/fastquant.git enzoampil/fastquant
0.01s
R-session-info
R session information
No DESCRIPTION file found, user must supply their own install and script steps
```

Any thoughts?

**Comment by jgendrinal** [comment]
Will change the Travis CI file

**Comment by jgendrinal** [comment]
@enzoampil, R tests will still fail because Travis CI treats warnings and notes like errors, when they are generally negligible.

**Comment by enzoampil** [comment]
@jgendrinal So how do we properly run R tests from Travis? How is prophet doing this?

**Comment by jgendrinal** [comment]
@enzoampil prophet only builds the /python subdirectory. I'll make one more change to the .travis.yml to see if it will work.

**Comment by enzoampil** [comment]
@jgendrinal another thing, I think best to move the `examples` and `lessons` folders back to the root directory. Also, I noticed the  the `lessons` folder got deleted.

File: .travis.yml [hunk 1]
```diff
@@ -1,2 +1,13 @@
-language: python
-script: pytest tests/
+matrix:
+  include:
+    - language: python
+      before_script:
```
Comment (enzoampil) [inline .travis.yml:4]: Can we add the following before `before_script`:

```
      install:
        - pip install --upgrade pip
        - pip install -U -r python/requirements.txt
```

Comment (enzoampil) [reply .travis.yml:4]: This will make the python test pass

**Comment by enzoampil** [comment]
Also, is it just me or did the PR switch from the `r-dev` branch to your fork's master

File: .travis.yml [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+        - cd python
+      script:
+        - pytest tests
+    - language: r
```
Comment (enzoampil) [inline .travis.yml:8]: I think we should take out `R` coverage from TravisCI first. We can include it in a separate PR later. Let's focus on merging this

**Comment by jgendrinal** [comment]
Oh. I'm intentionally merging from master to master this time.

**Comment by enzoampil** [comment]
awesome, I’ll do a last QA pass on the diff and from there LGTM to merge

File: LICENSE [hunk 3]
```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```
Comment (enzoampil) [inline LICENSE:1]: @jgendrinal Can we add this licence back to the python directory pls? 😄

Comment (enzoampil) [reply LICENSE:1]: Nevermind can just fix this after merging

**Pull request merged**